src/client/components/SocialButtons.tsx

```diff
@@ -17,9 +17,9 @@ const containerStyles = css`
   ${from.mobileLandscape} {
     flex-direction: row;
   }
-  justify-content: center;
-  margin-top: ${space[12]}px;
-  margin-bottom: ${space[12]}px;
+  justify-content: space-between;
```

Comment:
Makes sure that social buttons are spread evenly across the whole span of the containing element as per designs